python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -427,6 +435,9 @@ async def do_move(self, data: salobj.BaseMsgType) -> None:
         """Go to the position specified by the most recent ``positionSet``
         command.
         """
+
+        self.assert_is_motion_locked()
```

Comment:
Isn't it a bit weird that you "assert motion is locked" to allow it to move? consider renaming this to something like `assert_not_locked` or something on those lines.

Reply:
Renamed in the commit: [b4e39d1](https://github.com/lsst-ts/ts_mtrotator/pull/97/commits/b4e39d11dc643201ed9fc28fd9b89b0c45c65069). Thanks!